dds/DCPS/transport/rtps_udp/RtpsUdpTransport.cpp

```diff
@@ -290,6 +314,17 @@ RtpsUdpTransport::configure_i(RtpsUdpInst& config)
                       false);
   }
 
+#if defined (ACE_RECVPKTINFO)
+  int sockopt = 1;
+  if (unicast_socket_.set_option(IPPROTO_IP, ACE_RECVPKTINFO, &sockopt, sizeof sockopt) == -1) {
+    ACE_ERROR_RETURN((LM_ERROR,
+                      ACE_TEXT("(%P|%t) ERROR: ")
+                      ACE_TEXT("RtpsUdpTransport::configure_i: setcokopt:")
```

Comment:
Typo here, perhaps should just be `set_option`

Reply:
Fixed.